code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -228,22 +245,22 @@
 		if(loc == target || loc == get_turf(target))
 			if(check_bot(target))	//Target is not defined at the parent
 				shuffle = TRUE
-				if(prob(50))	//50% chance to still try to repair so we dont end up with 2 floorbots failing to fix the last breach
+				if(prob(50)) //50% chance to still try to repair so we dont end up with 2 floorbots failing to fix the last breach
```

Comment:
Why not just check if the other bot `mode` is REPAIRING?

Reply:
Idk, that not part of my changes